sympy/physics/optics/polarization.py

```diff
@@ -86,50 +86,41 @@
 
 
 def jones_vector(psi, chi):
-    u"""A Jones vector corresponding to a polarization ellipse with `psi` tilt,
-    and `chi` circularity.
-
-    Parameters
-    ----------
-    ``psi`` : numeric type or sympy Symbol
-        The tilt of the polarization relative to the `x` axis.
-
-    ``chi`` : numeric type or sympy Symbol
-        The angle adjacent to the mayor axis of the polarization ellipse.
-
-
-    Returns
-    -------
-    Matrix
-        A Jones vector.
+    u"""
+    A Jones vector corresponding to a polarization ellipse with ``psi`` tilt,
+    and ``chi`` circularity.
 
     Examples
-    --------
-    The axes on the Poincaré sphere.
+    ========
```

Comment:
Note that SymPy's docstyle deviates from the numpydoc style in this respect.

From the [Documentation Style Guide](https://docs.sympy.org/latest/documentation-style-guide.html#docstring-sections):
> "Headings should be underlined with the same length in equals signs."

From the [numpydoc Docstring Guide](https://numpydoc.readthedocs.io/en/latest/format.html#sections):
> "Each heading should be underlined in hyphens..."

It might be valuable to note this deviation in the documentation style guide.